src/game/Tactical/DisplayCover.cc

```diff
@@ -329,7 +329,7 @@ static INT8 CalcCoverForGridNoBasedOnTeamKnownEnemies(SOLDIERTYPE const* const p
 	{
 		bPercentCoverForGridno = iTotalCoverPoints / bNumEnemies;
 		INT32 const iTemp = bPercentCoverForGridno - (iHighestValue / bNumEnemies) + iHighestValue;
-		bPercentCoverForGridno = std::min(0, 100 - iTemp);
+		bPercentCoverForGridno = std::min(0, std::clamp(100 - iTemp, 0, 100));
```

Comment:
This will always be 0, no?

Either go with a working clamp:
```suggestion
		bPercentCoverForGridno = std::clamp(100 - iTemp, 0, 100);
```

Or like it was before:
```suggestion
		bPercentCoverForGridno = 100 - std::min(iTemp, 100);
```

Reply:
Total mixup on the logic there, my bad.